src/modules/translation/languages/en.json

```diff
@@ -491,6 +492,21 @@
     "total_land": "Total Land",
     "empty_land": "Empty Land"
   },
+  "land_ens_page" : {
+    "title": "Set Name",
+    "subtitle": "Set the ethereum name service to your land",
+    "select_names": {
+      "placeholder": "Your registered names",
+      "message": {
+        "choose_name": "Select a name for this land",
+        "name_available": "This name will work for your land",
+        "name_assigned": "This name has been assigned to this land"
+      }
+    },
+    "submit_message": {
+      "success": "Name set properly to your land"
```

Comment:
@nachomazzara @cazala I'm confused but should this `land` be `Land` or `LAND` or neither?
Also, should we rename it on the Spanish translations? (it says `tierra` in multiple places)

Reply:
Since the introduction of the Land tab in the builder we started using "Land" or "land" when we are taking about either parcels or estates. We are not using LAND anymore when talking about individual parcels, we use "Parcel". We use LAND when we talk about the token. That's as far as I understand it.